guess_number/guess.py

```diff
@@ -0,0 +1,71 @@
+import random
+
+random_number = random.randint(0, 100)
+
+
+def guess_number():
+
+    while True:
+        user_guess = int(input("Guess a number: "))
+        if user_guess == random_number:
+            print("Just right!")
+            return random_number
+            break
+
+        elif user_guess < random_number:
+            print("Too low")
+
+        elif user_guess > random_number:
+            print("Too high")
+
+
+print(guess_number())
```

Comment:
You should be able to just call `guess_number()` without the `print` surrounding it because the function itself will print. This is basically redundant.